README.md

```diff
@@ -35,15 +35,15 @@ this adapter using Helm.
 
 To deploy this adapter, you will need a cluster with the following minimum setup.
 
-* Kubernetes v1.10.0
-* Istio v1.0.3
+* Kubernetes v1.15.0
+* Istio v1.4, v1.5
```

Comment:
nit: this can just say v1.4.0

Reply:
I guess i did this based on previous review comment.

Reply:
could you paste the previous comment here, i tried to find a previously resolved comment about this topic and couldn't find one

Reply:
https://github.com/vmware/wavefront-adapter-for-istio/pull/94#discussion_r419572605, and also in the slack @venilnoronha suggested to support 1.5 ISTIO as well.

Reply:
Thanks. This was a nit anyways, keeping it is also fine with me.